Sources/DesignResourcesKit/FontViewModifiers.swift

```diff
@@ -0,0 +1,158 @@
+//
+//  FontViewModifiers.swift
+//
+//  Copyright © 2023 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+import SwiftUI
+
+#if canImport(UIKit)
```

Comment:
No, I don't see much point in conditionally importing swiftUI, since any thing that uses DRK must surely be dependant on swiftui anyway.

As is I need the UIKit references for UIFont, so I suppose this is arguably pointless, since I've needlessly made the whole thing depend on UIKit. I could make equivalent methods for Font as well as UIFont, but not sure it's worthwhile until we know what we're doing with macOS

Reply:
👍